core/src/main/scala-3/com/itv/scheduler/extruder/DeriveSyntax.scala

```diff
@@ -0,0 +1,26 @@
+package com.itv.scheduler
+package extruder
+
+import cats.data.Chain
+import com.itv.scheduler.{JobDataEncoder, JobDecoder}
+import shapeless3.deriving.*
+
+trait DeriveSyntax {
+//  given deriveJobDecoder[A](implicit ev: => DerivedJobDecoder[A]): JobDecoder[A]             = ev
+//  def deriveJobDataEncoder[A](implicit ev: => DerivedJobDataEncoder[A]): JobDataEncoder[A] = ev
+  implicit def deriveJobCodec[A](implicit ev1: => DerivedJobDecoder[A], ev2: => DerivedJobDataEncoder[A]): JobCodec[A] =
```

Comment:
I think this seems to be the method that doesn't work, in that it can't derive the decoder and encoder